tests/test_single_document.py

```diff
@@ -253,6 +253,32 @@ def test_implementation(self) -> 'Generator':
     def test_implementation_location_link(self) -> 'Generator':
         yield from self.__run_goto_test(GOTO_RESPONSE_LOCATION_LINK, 'implementation', 'implementation')
 
+    def test_expand_selection(self) -> 'Generator':
+        self.insert_characters("abcba\nabcba\nabcba\n")
+        self.view.run_command("lsp_selection_set", {"regions": [(2, 2)]})
+        self.assertEqual(len(self.view.sel()), 1)
+        self.assertEqual(self.view.substr(self.view.sel()[0]), "")
+        self.assertEqual(self.view.substr(self.view.sel()[0].a), "c")
+        response = [{
+            "parent": {
+                "parent": {
+                    "range": {"start": {"line": 0, "character": 0}, "end": {"line": 0, "character": 5}}
+                },
+                "range": {"start": {"line": 0, "character": 1}, "end": {"line": 0, "character": 4}}
+            },
+            "range": {"start": {"line": 0, "character": 2}, "end": {"line": 0, "character": 3}}
+        }]
```

Comment:
Can you also test expanding only "on one side"? So, for example, "start" stays the same but "ends" moves forward.

Maybe also test expanding across the lines?

Reply:
> Can you also test expanding only "on one side"? So, for example, "start" stays the same but "ends" moves forward.

Done.

> Maybe also test expanding across the lines?

Doesn't matter, I'm using sublime.Region which doesn't work with the notion of lines altogether.